### PR TITLE
Angela and Jay Sass Assignment

### PR DESCRIPTION
**Pull request by AngelaDiaFazio**

### CODE REVIEW

File: Angela-Jay-Sass/css/app.css [hunk 1]
```diff
@@ -0,0 +1,266 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (blackfalcon) [inline Angela-Jay-Sass/css/app.css]: ignoring doesn't remove the file. Delete the file and commit that delete.

File: Angela-Jay-Sass/css/app.css.map [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (blackfalcon) [inline Angela-Jay-Sass/css/app.css.map]: same here:

`git add --all` > `git commit -m "remove files"` > `git push`

File: Angela-Jay-Sass/sass/_colors.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.bkg-color {
+  background-image: url(../fabric_plaid.png);
```
Comment (blackfalcon) [inline Angela-Jay-Sass/sass/_colors.scss]: relative paths are bad.

File: Angela-Jay-Sass/sass/_colors.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  background-color: $body-light;
+  background-blend-mode: multiply;
+}
+.bkg-color-feature-2 {
+  background-image: url(../fabric_plaid.png);
```
Comment (blackfalcon) [inline Angela-Jay-Sass/sass/_colors.scss]: same here

File: Angela-Jay-Sass/sass/_config.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+$primary-color: #2EE0AE; //bright green
+$secondary-color: #C7E2C3; //pastel green
+$tertiary-color: #664C4B; //walnut
+$base-font-dark-color: #434545; // deep grey
+$base-font-light-color: #FAFEFF; // off white
+$body-light: #F8F4C7; //beige
+$body-dark: #E3D0A6; //camel
+$link: #664C4B; //walnut
+// Mixin
+@mixin large-text {
```
Comment (blackfalcon) [inline Angela-Jay-Sass/sass/_config.scss]: mixins are not config code

File: Angela-Jay-Sass/sass/_config.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  font: {
+    family: $font-base;
+    size: 5em;
+    weight: bold;
+  }
+  color: $tertiary-color;
+}
+// Placeholder
+%aside-padding {
```
Comment (blackfalcon) [inline Angela-Jay-Sass/sass/_config.scss]: placeholders are not config code

File: Angela-Jay-Sass/sass/_type.scss [hunk 7]
```diff
@@ -0,0 +1,17 @@
+h1 {
+  font: 100% $font-stack;
+  color: $primary-color;
+  p {
```
Comment (blackfalcon) [inline Angela-Jay-Sass/sass/_type.scss]: a `p` nested inside a `h1`, why?

**Comment by blackfalcon** [comment]
please update or I will grade as is

**Comment by AngelaDiaFazio** [comment]
Sorry, Jay thought it was already graded. I will meet with him today.

On Tue, Oct 21, 2014 at 3:23 PM, blackfalcon notifications@github.com
wrote:

> please update or I will grade as is
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/SEA-Design-Dev/sass-101/pull/7#issuecomment-60008869.

## 

ADF

File: Angela-Jay-Sass/css/app.css (hunk 1 above)
Comment (Jay-TechSalon) [inline Angela-Jay-Sass/css/app.css]: Done

File: Angela-Jay-Sass/sass/_colors.scss (hunk 4 above)
Comment (Jay-TechSalon) [inline Angela-Jay-Sass/sass/_colors.scss]: images are placed in an image folder

File: Angela-Jay-Sass/sass/_config.scss (hunk 6 above)
Comment (Jay-TechSalon) [inline Angela-Jay-Sass/sass/_config.scss]: moved both of these not config code to proper files they would be with, type, and layout respectively

File: Angela-Jay-Sass/sass/_type.scss (hunk 7 above)
Comment (Jay-TechSalon) [inline Angela-Jay-Sass/sass/_type.scss]: no reason there!

**Comment by Jay-TechSalon** [comment]
All set and fixed up as asked!

**Pull request closed without merging**